test/unittests/execute_control_test.cpp

```diff
@@ -195,14 +192,10 @@ TEST(execute_control, nested_blocks)
       i32.const 64
       i32.or)
     */
-
-    auto bin = from_hex(
-        "0061736d010000000105016000017f030201000a41013f01017f"
-        "0240200041017221000240200041027221000240200041047221000c"
-        "77"  // <-- br's immediate value.
-        "200041087221000b200041107221000b200041207221000b200041c000720b000c046e"
-        "616d6502050100010000");
-    constexpr auto br_imm_offset = 54;
+    auto wasm = from_hex(
```

Comment:
Here the `const` is omitted and the script will leave it as it when updating.

Reply:
Why is that needed if you have the comment above with `--no-check`? That is used to ensure the example still compiles, but the proper bytecode is actually different (to fill in the imm)?

Instead of all this mess why not have 2-3 examples with different immediates?

Reply:
Added 4 examples.